wagtail/contrib/search_promotions/models.py

```diff
@@ -1,3 +1,10 @@
+"""
+# Search Promotions models
+
+## To do:
+    - Eliminate duplicates of Query and QueryDailyHits classes in wagtail.search
+"""
+
```

Comment:
No need to add this docstring at the top, we don't really do this. 

If there is something else to improve, best to just raise a new issue. Todo comments tend to get lost in the noise.